client/src/app/site/common/components/search/search.component.html

```diff
@@ -1,58 +1,72 @@
 <os-head-bar [nav]="false" [goBack]="true">
     <!-- Title -->
     <div class="title-slot"><h2 translate>Search results</h2></div>
+
     <!-- Menu -->
     <div class="menu-slot">
         <button type="button" mat-icon-button [matMenuTriggerFor]="menu"><mat-icon>more_vert</mat-icon></button>
     </div>
 </os-head-bar>
 
 <!-- search-field -->
-<div class="search-field">
-    <form [formGroup]="quickSearchform">
-        <mat-form-field>
-            <input matInput osAutofocus formControlName="query" (keyup)="quickSearch()" />
+<div class="search-container">
+    <div class="search-field">
+        <mat-form-field appearance="outline" class="search-component">
+            <mat-label>{{ 'Search' | translate }}</mat-label>
+            <input #searchBox matInput osAutofocus (input)="quickSearch(searchBox.value)" />
+            <button mat-icon-button matPrefix (click)="quickSearch('')">
+                <mat-icon>close</mat-icon>
+            </button>
             <mat-icon matSuffix>search</mat-icon>
         </mat-form-field>
-    </form>
-</div>
 
-<mat-menu #menu="matMenu">
-    <button mat-menu-item *ngFor="let registeredModel of registeredModels" (click)="toggleModel(registeredModel)">
-        <mat-checkbox [checked]="registeredModel.enabled" (click)="$event.preventDefault()">
-            <span>{{ registeredModel.verboseNamePlural | translate }}</span>
-        </mat-checkbox>
-    </button>
-</mat-menu>
+        <mat-form-field class="search-sort">
```

Comment:
looks a little bit odd. Better put either in the page-menu, which could be extended anyways.

- change `toggleModel` to pass a click-event, use `event.stopPropagation();` to leave the menu open after klick
- add `Sort by ID` and `Sort by title` to the menu.
- Use a `mat-divider` to separate the sorting from the rest of the content